google/cloud/log.h

```diff
@@ -147,10 +147,12 @@ inline namespace GOOGLE_CLOUD_CPP_NS {
  *
  *
  */
-#define GCP_LOG(level) GOOGLE_CLOUD_CPP_LOG_I(level, LogSink::Instance())
+#define GCP_LOG(level)                                               \
+  GOOGLE_CLOUD_CPP_LOG_I(GOOGLE_CLOUD_CPP_PP_CONCAT(GCP_LS_, level), \
+                         ::google::cloud::LogSink::Instance())
 
 #ifndef GOOGLE_CLOUD_CPP_LOGGING_MIN_SEVERITY_ENABLED
-#define GOOGLE_CLOUD_CPP_LOGGING_MIN_SEVERITY_ENABLED NOTICE
+#define GOOGLE_CLOUD_CPP_LOGGING_MIN_SEVERITY_ENABLED GCP_LS_DEBUG
```

Comment:
Shouldn't this be GCP_LS_NOTICE?

Reply:
That would disable the new messages at compile-time, which would make this PR kind of boring.

So we can do a few things:

- Make the new messages "NOTICE" severity. I did not think that logging traffic should be that high (I think google practice is to use "INFO" for traffic anyway).

- Make the new messages "INFO" and leave the minimum compile-time enable level at "NOTICE", that would make this PR somewhat uninteresting, we can change the level in a future PR.

- Also change the minimum compile-time level in this PR. I chose this. Partly because in the original PR we [discussed](https://github.com/GoogleCloudPlatform/google-cloud-cpp/pull/601#discussion_r192615443) what was the right minimum compile-time level but never reached a decision and NOTICE just won by default.  A mistake in hindsight.